locales/en.json

```diff
@@ -25,6 +25,9 @@
     "shouted": "Shouted",
     "commented": "Commented"
   },
+  "search": {
+    "placeholder": "Search …"
```

Comment:
```suggestion
    "placeholder": "Search…"
```

Reply:
I would suggest to remove the dots entirely @appinteractive .

My little investigation showed: Mostly there seems to be only a magnifying glas icon and no placeholder text. If there is a placeholder text, it’s without the dots. May because the dots can make the impression that the search is already in processing?!

Reply:
I agree, so let’s remove the dots. 👍

Reply:
From the usage perspective, I thought a bit about the search and the related api, and would go with a list under the field which shows some results for posts, people, tags etc. 

That way you can ether jump to a result or just press enter and land someware to a grown up search page. 

Just as a thing for a later version? I will do some experiment for the api side in the next sprint maybe.

Reply:
That's a good idea. 👍🏼

What I experienced often by using the alpha is, that I've seen a post, didn't saved the link and was just not able to find it again days later. Same with Facebook, except that one can save posts there.

I would love to have a kind of extended search. But should be easy to use and nice looking (not technical). For the future … ?
I have some ideas close to the macOS file search …

And I think HC users should be able to save posts, because post should be something where users can work on (see Can Do’s etc.) at the moment or days later. A future functionality maybe …

Just my impressions … and suggestions for discussion …

Martin invited me to the Scrum meeting next Monday. 😃

Reply:
Should we try to figure out the API for queryring the search by our selfs?
Or is it better if you prepare something what we can use?
I have a pairing with @mattwr18 right now on this issue …

Reply:
I have studied the progress of the api library we use for the last months and have some ideas how we can setup the api for that usage. I would just try to give my ideas and maybe draft an interface which we can discuss. But if anyone have ideas or input, go for it and let’s discuss!

Reply:
Okay. That seems to be a good idea to me. Than we learn at the end a lot by discussing your draft, but don’t have to study, what you have already done. 😉

Reply:
@mattwr18 and I have figured out in more detail, what „ds-input“ is able to do @appinteractive :

- But we were not able to extent it by `@keyup.enter="onEnter“` and get `onEnter()` to work. Is there any trick to do so? Did it really work in the Alpha? Do we like to have this function in general since 700 milliseconds are not much time to wait for a search?
- We like to show a „clear all text icon“ on the right side, if a text is in the search field. We could not figure out what the name may be if it’s already existent?

Reply:
Try `@keyup.native.enter` or similar. Take a look at the location input in your settings.

Reply:
I have a simple fulltext search working already for posts. Will create a wip pullrequest later today so you can work with it already.